ios/brave-ios/Sources/BraveWallet/Crypto/Stores/NFTDetailStore.swift

```diff
@@ -142,9 +144,7 @@ class NFTDetailStore: ObservableObject, WalletObserverStore {
         networkInfo = network
       }
 
-      if owner == nil {
-        updateOwner()
-      }
+      updateOwner()
```

Comment:
Is this changed just because it's fetching from cache now instead of network?

Reply:
yes. also for the case when this NFT is being sent to another address. once the tx lands on chain, and user is still in the NFT detail screen, we want to refresh the balance and display the new owner address.